lec01-introduction.Rmd

```diff
@@ -2,3 +2,171 @@
+
+R Markdown can easily be converted into html and pdf documents with figures and professional typesetting. There are actually many academic papers that are written entirely in R Markdown and it is great for assignments or reports: I have used this myself for some of my progress updates to my supervisor. Since R Markdown converts to html, it is also easy to publish simple good-looking websites in it, for example the syllabus website for this course is made via an R Markdown template that Luke set up.
+
+Let's try to convert this new document. First, let's go through it and see what we expect the output to include. First we see this intro section here, this is where you specify the title of your document, what kind of output you want, and a few other things that we will try later. This is called the yaml-block, but that's not important for now. Then we have a heading, a link some bold text, and a few code blocks. OK, let's see what this looks like. To create the output document, we poetically say that we will knit our R Markdown into the HTML document. Luckily, it is much simpler than actually knitting something, just press this button here and the new document will be created.
+
+As you can see, the title showed up as we would expect, the lines with pound sign(s) in front of them were converted into headers and we can see both the code and its output! So the plots are generated directly in the report without us having to cut and paste images! If we change something in the code, we don't have to find the new images and paste it in again, the correct one will appear right in your code. E.g. if we realize that we actually used the wrong data set for the plot, and we want to plot `cars` instead, we just change `pressure` to `cars` and then click stitch again, done!
+
+Furthermore, if we want to change the style of our output document, it is quite easy to do so. Let's add a subheading first, then to keep track of things we want a table of contents, _knit_. Now we realize that we want to include numbers in the report to keep with the departmental guidelines, _knit_. We could alter the style of our report by adding a theme, _knit_. And finally we remember that we need to hand in our assignments as pdf-files, so let's create a pdf instead of a html-document, _remove themee and knit_.
+
+```yaml
+---
+title: "HTML output demo"
+output:
+  html_document:
+    toc: true
+    number_section: true
+    theme: united
+---
+```
+## R Notebook interactivity
```

Comment:
I would actually talk about this first, maybe even instead of RMarkdown.

Reply:
This fits in with your other comment about notebooks.